.github/workflows/client-angular.yaml

```diff
@@ -3,30 +3,59 @@ name: Client Angular
 on: [push]
 
 jobs:
-  test:
-
+  ng-test:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
 
     - name: Cache node modules
-      uses: actions/cache@v1
+      uses: actions/cache@v2
       with:
         path: ~/.npm
         key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
         restore-keys: |
           ${{ runner.os }}-node-
 
-    - name: Use Node.js 12.x
+    - name: Use Node.js 14
       uses: actions/setup-node@v1
       with:
-        node-version: 12.x
+        node-version: 14
```

Comment:
Is there a difference between `14` and `14.x` (which is what you use in the `ng-lint` setup below)?

Reply:
I don't think so but I'll make them all just `14` and also update to `actions/setup-node@v2`.